binaryen.opam

```
@@ -18,5 +18,5 @@ depends: [
   "js_of_ocaml" {>= "3.10.0"}
   "js_of_ocaml-ppx" {>= "3.10.0"}
   "js_of_ocaml-compiler" {>= "3.10.0"}
-  "libbinaryen" {>= "101.0.1" < "102.0.0"}
+  "libbinaryen" {>= "102.0.0" < "103.0.0"}
```

```suggestion
  "libbinaryen" {>= "102.0.1" < "103.0.0"}
```
We need https://github.com/ocaml/opam-repository/pull/20455 because the wasm-delegations.def file was missing.